oasislmf/preparation/gul_inputs.py

```diff
@@ -363,39 +355,29 @@ def get_gul_input_items(
     # If the group id is set according to the correlation group field then map this field
     # directly, otherwise create an index of the group id fields
 
-    # Keep group_id consistance by lower casing column names and sorting
-    group_id_cols.sort()
-    group_id_cols_list = sorted([c.lower() for c in group_id_cols])  # list of case insessative column names
-    group_id_cols_map = {c: c.lower() for c in group_id_cols}         # mapping from PascalCase -> 'lower_case'
+    # keep group_id consistance by lower casing column names and sorting
+    damage_group_id_cols_map = {c: c.lower() for c in sorted(damage_group_id_cols)}        # mapping from PascalCase -> 'lower_case'
+    hazard_group_id_cols_map = {c: c.lower() for c in sorted(hazard_group_id_cols)}        # mapping from PascalCase -> 'lower_case'
 
     if correlation_check is True:
         gul_inputs_df['group_id'] = gul_inputs_df[correlation_group_id]
 
-    elif hashed_group_id is False:
-
-        if len(group_id_cols) > 1:
-            gul_inputs_df['group_id'] = factorize_ndarray(
-                gul_inputs_df.loc[:, group_id_cols].values,
-                col_idxs=range(len(group_id_cols)),
-                sort_opt=True
-            )[0]
-
-        else:
-            gul_inputs_df['group_id'] = factorize_array(
-                gul_inputs_df[group_id_cols[0]].values
-            )[0]
+    # TODO: should we have damage_correlations_check and hazard_correlations_check?
```

Comment:
We can discuss or TODO usage but to me, we should avoid TODO in code.
If there is some left over work I prefer if we create an issue for it